buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/project-groups.kt

```diff
@@ -58,8 +58,8 @@ object ProjectGroups {
             rootProject.project(":processServices"),
             rootProject.project(":resources"))
 
-    val Project.projectsRequiringJava8
-        get() = setOf(rootProject.project(":testingJunitPlatform"))
+    val Project.projectsRequiringLanguageLevel8
```

Comment:
I added `buildScanPerformance` here because it's the only one which depends on `internalPerformanceTesting`.

Reply:
What about the `performance` project? I think the dependency is added here: https://github.com/gradle/gradle/blob/0b7c9abce9076ceca665fc894b6e36a84547309c/buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt#L150

Reply:
Strange, because currently this branch work in my IDEA - I can run tests in `InternalPerformanceTesting` and `performanceTest` in `performance.

Reply:
Very strange...